fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaHomeFragment.java

```diff
@@ -11,6 +11,8 @@
 
 import com.github.dedis.popstellar.R;
 import com.github.dedis.popstellar.databinding.SocialMediaHomeFragmentBinding;
+import com.github.dedis.popstellar.utility.error.ErrorUtils;
+import com.github.dedis.popstellar.utility.error.UnknownLaoException;
```

Comment:
I've  just realised that I read it like `unknown LaoException` (as a generic exception) instead of `UnknownLao exception`. I guess a few students are going to be confused as well. Naming is so challenging... how about BadLaoIdException or CannotFindLaoException or ...?

Reply:
Oh I did not realise the source of confusion 😅. `NoSuchLaoException`  ?
The advantage of the current one is that it is consistent with the current naming of exceptions.
Moreover if changing it we must, I think it should be done in  a separate PR as the class was not introduced here

Reply:
Regardless I've added a comment for clarification